doc/src/learn/sui-compared.md

```diff
@@ -123,6 +123,20 @@ Sui mitigates this with checkpoints. A checkpoint is established every time an i
 
 Sui uses the state commitment that arrives upon epoch change. Sui requires a single answer from the multiple validators and leverages an accessory protocol to derive the hash representing the state of the blockchain. This protocol consumes little bandwidth and does not impede the ingestion of transactions. Validators produce checkpoints at every epoch change. Sui requires the validators to also produce checkpoints even more frequently. So users may use these checkpoints to audit the blockchain with some effort.
 
+## Summary
+
+|                 **Concept**                 |                  **Sui**                  |  **Traditional blockchain** |
+|:-------------------------------------------:|:-----------------------------------------:|:---------------------------:|
+| _Unit of storage_                           | object                                    | account                     |
```

Comment:
Object goes a bit beyond the unit of storage in that all smart contracts explicitly take objects as inputs and objects so it is also semantically the fundamental unit of change and operation for the distributed ledger (calling it unit of storage may be close enough and is not inaccurate, but it may be a bit too generic and doesn't underscore the fact that it is the unit of consensus).  Account however is not a unit of storage?